CourseDevelopment/Week1/Chapter1.MD

```diff
@@ -1,129 +1,80 @@
-# Course Development Week 1
-Course Development Week 1 for the Open Science Course for physics students at graduate level. This document is based on the Syllabus.MD.
+# Open Science for Physicists, Lecture 1
 
-Date: Nov/02/2022
++ Last updated Date: 05/06/2022
++ Lecturer: @SanliFaez 
 
-Name: Hendrik Snijder @hendelhendel
+#### Contributors: 
++ Erik van Sebille @erikvansebille
++ Hendrik Snijder @hendelhendel
 
-Open tasks @SanliFaez @erikvansebille
-+ Feedback on lecture-tutorial distribution of topics
-+ Feedback on lecture format
-+ Feedback on tutorial format
+![Frontpage](./Week1/Chapter1.png?raw=true)
```

Comment:
Figure does not work at https://github.com/SanliFaez/OS4Physicists/blob/origin/chapters1and2/CourseDevelopment/Week1/Chapter1.MD